guides/v2.3/graphql/reference/categories.md

```diff
@@ -40,17 +40,19 @@ Attribute | Data type | Description
 `product_count`| Int | The number of products in the category
 `default_sort_by`| String | The attribute to use for sorting
 `products(<attributes>)` | `CategoryProducts` | The list of products assigned to the category
-`children` | `CategoryTree` |
+`breadcrumbs` | `Breadcrumb` |
+`children` | `CategoryTree` | A `CategoryTree` object that contains information about a child category. You can specify up to 10 levels of child categories.
+
+
+#### CategoryProducts object
 
 The `products` attribute can contain the following attributes:
 
 Attribute | Data type | Description
 --- | --- | ---
 `pageSize` | Int | Specifies the maximum number of results to return at once. This attribute is optional. The default value is 20.
 `currentPage` | Int |  Specifies which page of results to return. The default value is 1.
-`sort` | `ProductSortInput` Specifies which attribute to sort on, and whether to return the results in ascending or descending order. See [Searches and pagination in GraphQL]({{ page.baseurl }}/graphql/search-pagination.html) for more information.
-
-#### CategoryProducts
+`sort` | `ProductSortInput` | Specifies which attribute to sort on, and whether to return the results in ascending or descending order. See [Searches and pagination in GraphQL]({{ page.baseurl }}/graphql/search-pagination.html) for more information.
```

Comment:
Consider dropping "for more information"

Reply:
Changed to [Searches and pagination in GraphQL] describes sort orders.